Zigbee2MqttAssistant/Services/BridgeStateService.cs

```diff
@@ -87,6 +87,7 @@ public ZigbeeDevice UpdateDevice(string friendlyName, string jsonPayload, out bo
 			var json = JObject.Parse(jsonPayload);
 			var linkQuality = json["linkquality"]?.Value<ushort>();
 			var battery = json["battery"]?.Value<decimal>();
+			var updateAvailable = json["update_available"]?.Value<bool?>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'updateAvailable'.](https://app.codacy.com/gh/yllibed/Zigbee2MqttAssistant/pullRequest?prid=5086977)